fds/run.py

```diff
@@ -44,6 +46,27 @@ def pre_execute_hook(self):
             self.printer.error("git executable is not found, please install git from https://git-scm.com/downloads")
             ret_code |= 2
 
+        r = requests.get("https://pypi.python.org/pypi/fastds/json")
+        data = r.json()
+        latest_version = data["info"]["version"]
+        if latest_version != __version__:
+            questions = [
+                {
+                    'type': 'confirm',
+                    'message': "You are using fds version " + __version__ + 
+                                ", however version " + latest_version + " is available. Should we upgrade using `pip install fastds --upgrade`",
```

Comment:
One minor suggestion as we use f-string for formatting.
```suggestion
                    'message': f"You are using fds version {__version__}, however version {latest_version} is available. Should we upgrade using `pip install fastds --upgrade`",
```